CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ### Bug fixes
 
+* [#6027](https://github.com/rubocop-hq/rubocop/issues/6027): Fix `Style/IpAddresses` choking on an empty string. ([@dvandersluis][])
```

Comment:
The cop has not yet been released, so this entry is not needed.

Reply:
Yeah I wasn't sure about that, removed it now.